[2025-04-09T16:04:24.893+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T16:04:24.903+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2025-04-09T16:04:20.132214+00:00 [queued]>
[2025-04-09T16:04:24.909+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.is_weather_api_ready manual__2025-04-09T16:04:20.132214+00:00 [queued]>
[2025-04-09T16:04:24.910+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 3
[2025-04-09T16:04:24.927+0000] {taskinstance.py:2890} INFO - Executing <Task(HttpSensor): is_weather_api_ready> on 2025-04-09 16:04:20.132214+00:00
[2025-04-09T16:04:24.936+0000] {standard_task_runner.py:72} INFO - Started process 31241 to run task
[2025-04-09T16:04:24.940+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'is_weather_api_ready', 'manual__2025-04-09T16:04:20.132214+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpsxjq98p3']
[2025-04-09T16:04:24.941+0000] {standard_task_runner.py:105} INFO - Job 77: Subtask is_weather_api_ready
[2025-04-09T16:04:25.022+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.is_weather_api_ready manual__2025-04-09T16:04:20.132214+00:00 [running]> on host ip-172-31-10-226.eu-north-1.compute.internal
[2025-04-09T16:04:25.093+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='myemail@domain.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='is_weather_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T16:04:20.132214+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T16:04:20.132214+00:00'
[2025-04-09T16:04:25.095+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-09T16:04:25.095+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-09T16:04:25.095+0000] {logging_mixin.py:190} INFO - Current task name:is_weather_api_ready state:running start_date:2025-04-09 16:04:24.903996+00:00
[2025-04-09T16:04:25.095+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag and current dag run status:running
[2025-04-09T16:04:25.095+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T16:04:25.096+0000] {baseoperator.py:423} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2025-04-09T16:04:25.096+0000] {http.py:145} INFO - Poking: /data/2.5/weather?q=Portland&APPID=4d13f07efe491003cd53c704f189b00f
[2025-04-09T16:04:25.102+0000] {base.py:84} INFO - Retrieving connection 'weathermap_api'
[2025-04-09T16:04:25.194+0000] {base.py:339} INFO - Success criteria met. Exiting.
[2025-04-09T16:04:25.201+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T16:04:25.201+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=is_weather_api_ready, run_id=manual__2025-04-09T16:04:20.132214+00:00, execution_date=20250409T160420, start_date=20250409T160424, end_date=20250409T160425
[2025-04-09T16:04:25.224+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-09T16:04:25.224+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-09T16:04:25.225+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag queued_at:2025-04-09 16:04:20.154559+00:00
[2025-04-09T16:04:25.225+0000] {logging_mixin.py:190} INFO - Task hostname:ip-172-31-10-226.eu-north-1.compute.internal operator:HttpSensor
[2025-04-09T16:04:25.238+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T16:04:25.297+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T16:04:25.304+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
